src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] ballArray = new Ball[3];
+
+        for (int i = 0; i < ballArray.length; i++) {
+            ballArray[i] = lottery.getRandomBall();
+            System.out.println(ballArray[i].toString());
+        }
     }
```

Comment:
we don't need an array, call print on the object returned by getRandomBall method rightaway :)
toString is called on object being printed by default, it's redundant